content/pt-br/docs/reference/setup-tools/kubeadm/generated/kubeadm_certs_generate-csr.md

```diff
@@ -0,0 +1,96 @@
+<!--
+The file is auto-generated from the Go source code of the component using a generic
+[generator](https://github.com/kubernetes-sigs/reference-docs/). To learn how
+to generate the reference documentation, please read
+[Contributing to the reference documentation](/docs/contribute/generate-ref-docs/).
+To update the reference conent, please follow the 
+[Contributing upstream](/docs/contribute/generate-ref-docs/contribute-upstream/)
+guide. You can file document formatting bugs against the
+[reference-docs](https://github.com/kubernetes-sigs/reference-docs/) project.
+-->
+
+
+Gerar chaves e solicitações de assinatura de certificados
+
+### Sinopse
+
+Gera as chaves e as solicitações de assinatura de certificados (CSRs) para todos os certificados necessários para executar a camada de gerenciamento. Este comando também gera os arquivos kubeconfig parciais com dados de chave privada no campo "users &gt; user &gt; client-key-data" e, para cada arquivo kubeconfig, um arquivo ".csr" que o acompanha é criado.
+
+Esse comando foi projetado para uso no [Kubeadm External CA Mode](https://kubernetes.io/docs/tasks/administer-cluster/kubeadm/kubeadm-certs/#external-ca-mode). Ele gera CSRs que você pode enviar à sua autoridade de certificação externa para assinatura.
+
+Os certificados PEM assinados e codificados devem ser salvos ao lado dos arquivos da chave, usando ".crt" como extensão de arquivo ou, no caso de arquivos kubeconfig, o certificado assinado codificado PEM deve ser codificado em base64 e adicionado ao arquivo kubeconfig no campo "users &gt; user &gt; client-certificate-data".
+
+```
+kubeadm certs generate-csr [flags]
+```
+
+### Exemplos
+
+```
+  # O comando a seguir gerar as chaves e CSRs para todos os certificados do plano de controle e arquivos kubeconfig: 
```

Comment:
# O comando a seguir gera as chaves e CSRs para todos os certificados do plano de controle e arquivos kubeconfig:

an extra "r" in the word "gerar"